src/renderer/components/TranscriptionBlock.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled';
+import { Box, colors } from '@mui/material';
+import { Transcription } from 'sharedTypes';
+
+const TranscriptionBox = styled(Box)`
+  background: ${colors.common.white};
+  height: 100%;
+  padding: 20px;
+`;
+
+const Word = styled('span')`
+  &:hover {
+    background: orange;
+  }
+`;
+
+interface Props {
+  transcription: Transcription;
+  onWordClick: (wordIndex: number) => void;
+}
+
+const TranscriptionBlock = ({ onWordClick, transcription }: Props) => {
+  return (
+    <TranscriptionBox>
+      <p>
+        {/* eslint-disable-next-line prettier/prettier */}
+        {transcription.words.map((word, index) => (
```

Comment:
Nice work using the map. Instead of logging each word with a space next to it, maybe `join` the map result with a space

Reply:
Thats a smart idea and will fix my hover bug showing the space. Thanks!

Reply:
This is fixed now with the solution discussed over slack.